.dassie/Gemfile

```diff
@@ -64,3 +64,4 @@ gem 'riiif', '~> 2.1'
 gem 'rsolr', '>= 1.0', '< 3'
 gem 'sidekiq', '~> 6.0'
 gem 'twitter-typeahead-rails', '0.11.1.pre.corejavascript'
+gem 'valkyrie', '3.0.0.pre.beta.2'
```

Comment:
does this code depend on a 3.0.0 feature?

if so, i think we need to push for a 3.0.0 release soon.

Reply:
i'm trying to remember; it may have been your commit? https://github.com/samvera/valkyrie/commit/8f9277559201f4132bf67127ce915be97d771903

Reply:
oh, i don't think this PR's work depends on that. it's just an issue i ran into working on this.

let's roll back the dependency for now.

Reply:
sounds good; removed that and rebased